cart.rb

```diff
@@ -0,0 +1,57 @@
+require_relative 'bill'
+require 'terminal-table/import'
+
+class Cart
+
+  def initialize
+    @items = []
+  end
+
+  def add_item(grocery_item)
```

Comment:
Does `add_cart_item` sound better?

Reply:
Well, the class name was cart. Hence, add_item implies I am adding a cart item only.